mflowgen/glb_top/custom-init/outputs/floorplan.tcl

```diff
@@ -21,21 +21,23 @@ set core_margin_r [expr 5 * $horiz_pitch]
 set core_margin_l [expr 5 * $horiz_pitch]
 
 # Margins between glb tiles and core edge
-set tile_margin_t [expr 5 * $vert_pitch]
-set tile_margin_b [expr 120 * $vert_pitch]
-set tile_margin_l [expr 150 * $horiz_pitch]
-set tile_margin_r [expr 150 * $horiz_pitch]
+set tile_margin_t [expr 100 * $vert_pitch]
+set tile_margin_b [expr 50 * $vert_pitch]
+set tile_margin_l [expr 500 * $horiz_pitch]
+set tile_margin_r [expr 50 * $horiz_pitch]
 
 set tiles [get_cells *glb_tile*]
 set tile_width [dbGet [dbGet -p top.insts.name *glb_tile* -i 0].cell.size_x]
+# set tile_width_offset [expr 20 * $horiz_pitch]
+set tile_width_offset 0
```

Comment:
Does this param optionally add space between glb tiles rather than abutting? If so can we make the name clearer, or add a comment explaining that?